README.md

```diff
@@ -3,15 +3,15 @@
 
 # embroider-css-modules
 
-_CSS modules for Embroider + TypeScript projects_
+_CSS modules for Embroider projects_
```

Comment:
Definitely, `embroider-css-modules` works best when TypeScript and Glint are present. To avoid giving the impression that `embroider-css-modules` _requires_ TS, I listed TypeScript under the list of what it means to have a bleeding-edge Ember project.